presto-docs/src/main/sphinx/sql/call.rst

```diff
@@ -28,3 +28,6 @@ Call a procedure using named arguments::
 Call a procedure using a fully qualified name::
 
     CALL catalog.schema.test();
+
+.. note::
+    CALL statement is NOT used to invoke a DBMS style stored procedure.
```

Comment:
maybe "NOT used to invoke stored procedures that are defined on the target system (e.g., PostgreSQL database)".